<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>selectClass</class>
 <widget class="QMainWindow" name="selectClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>select</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="step_one_groupbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>411</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Step I:</string>
    </property>
    <widget class="QPushButton" name="select_folders_pushbutton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Select Folder</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clear_selected_folders_pushbutton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>25</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QLabel" name="selected_folders_label">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>15</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="step_two_groupbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>411</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Step II:</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Filter Options:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="mean_filter_checkbox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Mean</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="min_filter_checkbox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Min</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="max_filter_checkbox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Max</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="mean_shreshold_value_doublespinbox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>62</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="min_shreshold_value_doublespinbox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>62</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="max_shreshold_value_doublespinbox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>62</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Average Area</string>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Width:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Height:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>X Offset:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Y Offset:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="average_area_width_spinbox">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>42</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="average_area_height_spinbox">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>42</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="average_area_x_offset_spinbox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>42</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="average_area_y_offset_spinbox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>42</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="next_step_pushbutton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>260</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="previous_step_pushbutton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Prev.</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Step III:</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="process_bar">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="select.qrc"/>
 </resources>
 <connections/>
</ui>
